pkg/api/azenvtypes.go

```diff
@@ -165,7 +165,7 @@ var (
 		ImageOffer:     "aks-windows",
 		ImageSku:       "2019-datacenter-core-azurestack-ctrd",
 		ImagePublisher: "microsoft-aks",
-		ImageVersion:   "17763.4974.20232961",
+		ImageVersion:   "17763.5329.231230",
```

Comment:
This version should be “17763.5329.20240102” to match our existing pattern.

Reply:
Thanks, Updated.